File: app/src/main/java/com/exory550/exorypad/model/Defaults.java
package com.exory550.exorypad.model;

import java.util.Date;

public final class Defaults {
    public static final NoteMetadata metadata = new NoteMetadata(
        -1,
        "",
        new Date(),
        false
    );

    public static final NoteContents contents = new NoteContents(
        -1,
        null,
        null
    );

    public static final CrossRef crossRef = new CrossRef(
        -1,
        -1
    );

    private Defaults() {}
}

File: app/src/main/java/com/exory550/exorypad/model/Defaults.kt
package com.exory550.exorypad.model

import java.util.Date

object Defaults {
    val metadata = NoteMetadata(
        metadataId = -1,
        title = "",
        date = Date(),
        hasDraft = false
    )

    val contents = NoteContents(
        contentsId = -1,
        text = null,
        draftText = null
    )

    val crossRef = CrossRef(
        metadataId = -1,
        contentsId = -1
    )
}
